samples/pagination.js

```diff
@@ -28,17 +28,17 @@ async function main() {
     if (!request.pageToken) {
       result = {
         animals: ['cat', 'dog'],
-        nextPageToken: 'page2',
+        nextPageToken: 'page2'
```

Comment:
This isn't right.  I _think_ we want to keep the trailing comma here.  Prettier changed the default to include it in 2.0, so I removed the explicit config option in the prettierrc.  Maybe we need to add it backj?

Reply:
Done.